generate-offline-map/README.md

```diff
@@ -34,9 +34,8 @@ The map used in this sample shows the [stormwater network](https://arcgisruntime
 
 ## Additional information
 
-The creation of the offline map can be fine-tuned using [parameter overrides for feature layers](https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/generate-offline-map-overrides), or by using [local basemaps](https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/generate-offline-map-with-local-basemap)
-to achieve more customised results.
+The creation of the offline map can be fine-tuned using [parameter overrides for feature layers](https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/generate-offline-map-overrides), or by using [local basemaps](https://github.com/Esri/arcgis-runtime-samples-android/tree/master/java/generate-offline-map-with-local-basemap) to achieve more customised results. Also, this sample uses the GeoViewCompose Toolkit module to be able to implement a Composable MapView.
```

Comment:
We do not have these samples yet, let's remove the URLs pointing to the runtime repo and just have plain text here.  Add an issue to the backlog to update this section when the two samples are implemented in the future.